src/app/shutter/shutter.component.html

```diff
@@ -0,0 +1,10 @@
+<!-- BETA -->
+<cut-phase-bar phaseLabel="BETA" phaseLink="{{getSmartSurveyUrl()}}"></cut-phase-bar>
+
+<div class="container-fluid">
+  <div class="block-wrapper">
+    <div class="block mt-xl">
+      <h2 class="heading-h2">CCD is no longer in operation for your jurisdiction(s). You will be automatically redirected to Expert UI in {{redirectDelaySecs}} seconds.</h2>
```

Comment:
Users don't necessarily know CCD or Expert UI terms. Let's for now use a wording like below, which may further be changed by business later.
"This application is not available for your jurisdiction(s) on this address any more. You're being redirected to its new address, where it will be better supported with more features to come in the future."

Reply:
Addressed 👍